content/applications/hr/referrals.rst

```diff
@@ -776,3 +457,10 @@ spreadsheet from the drop-down menu, or select :guilabel:`New Spreadsheet`. Clic
 .. note::
    The spreadsheet is stored in the *Documents* application. This application needs to be installed
    in order to use the :guilabel:`Insert in Spreadsheet` option.
+
+.. toctree::
+
+   referrals/share_jobs
+
+.. seealso::
+   :doc:`referrals/share_jobs`
```

Comment:
Please swap the position of the seealso and toctree, so that the toctree is always at the end of the document :)
```suggestion

.. seealso::
   :doc:`referrals/share_jobs`

.. toctree::

   referrals/share_jobs

```